entries/s/soundcloud.com.json

```
@@ -1,12 +1,12 @@
 {
   "SoundCloud": {
     "domain": "soundcloud.com",
-    "contact": {
-      "facebook": "SoundCloud",
-      "twitter": "SCsupport"
-    },
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.soundcloud.com/hc/en-us/articles/20707203094043-Two-Factor-Authentication",
```

Just a minor cleanup of the URL :wink:
```suggestion
    "documentation": "https://help.soundcloud.com/hc/en-us/articles/20707203094043",
```